src/averageAges.js

```diff
@@ -15,7 +15,23 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  let onlyMen = people.filter((person) => person.sex === 'm');
+
+  function filterByCentury(man) {
+    return Math.ceil(man.died / 100) === century;
+  }
+
+  if (century) {
+    const menFilteredByCentury = onlyMen.filter(filterByCentury);
+
+    onlyMen = menFilteredByCentury;
+  }
```

Comment:
no need for creating additional function and variables if you can write it like this

```suggestion
  if (century) {
    onlyMen = onlyMen.filter(man => Math.ceil(man.died / 100) === century);
  }
```